README.md

```diff
@@ -1,3 +1,14 @@
+This project is officially end-of-life. Users are welcome to maintain it in a
```

Comment:
Minor: I find it clearer with a header like `# Deprecation Notice`, `# DEPRECATION NOTICE` or `# End of Life Notice` on top